src/resources/MachineLearning/MachineLearning.ts

```diff
@@ -42,6 +44,7 @@ export default class MachineLearning extends Resource {
         this.modelListing = new ModelListing(api, serverlessApi);
         this.relevanceGenerativeAnsweringConfig = new RelevanceGenerativeAnsweringConfiguration(api, serverlessApi);
         this.semanticEncoderConfig = new SemanticEncoderConfiguration(api, serverlessApi);
+        this.modelDetailedInfo = new ModelDetailedInfo(api, serverlessApi);
```

Comment:
Missing test coverage for this new property of the MachineLearning resource

Reply:
Fixed